app/domain/errors.rb

```diff
@@ -161,6 +166,11 @@ module Jwt
         code: "CONJ00015E"
       )
 
+      TokenFieldNotFoundOrEmpty = ::Util::TrackableErrorClass.new(
+        msg:  "Field '{0-field-name}' not found or empty in token",
```

Comment:
@shulifink please review

Reply:
fine

Reply:
is there any more information we give them to help them with moving on with this?

Reply:
nothing that i can think of